lecture_note.tex

```diff
@@ -3164,7 +3164,7 @@ \subsection{Gated Recurrent Units Address Vanishing Gradient}
 This is however not the case with the actual GRU or LSTM, because those update
 gates are {\em not} fixed but are adaptive with respect to the input. If the
 network detects that there is an important dependency being captured, the update
-gate will be closed ($u_j \approx 0$.) This will effectively strengthen the
+gate will be closed ($u_j \approx 0$). This will effectively strengthen the

```

Comment:
It's a usual convention to have the parenthesis include the punctuation mark immediately following the parentheses.